### DISCUSSION TITLE
[@types/ol] DataTile.d.ts file is not being generated

### DISCUSSION THREAD

**Discussion by ShantanuParode** [DefinitelyTyped/DefinitelyTyped#59846] [Issues with a @types package]
d.ts file is not being generated for ol/Source/DataTile.js. 
I was facing an issue when importing the file. It is importing the DataTile module from ol library file by default while other modules are getting imported from @types/ol. Please resolve ASAP or suggest any other method.

[a comment by an automated account is not shown]

**Comment by hanreev** [comment]
Hi @ShantanuParode
@types/ol only supports OpenLayers prior to version 6.6.0. As of version 6.6.0 OpenLayers has officially provided its own TypeScript definition. DataTile source is not yet available in version 6.5.0

Check out [@hanreev/types-ol](https://www.npmjs.com/package/@hanreev/types-ol) if you need @types/ol compatible definition for latest OpenLayers
